### PR TITLE
Improve the runserver script

### PR DESCRIPTION
**Pull request by pypingou**
This way the port and host where the application is ran can be set
via the CLI and a config file can be specified to adjust the local
app versus the default config.

### CODE REVIEW

File: runserver.py [hunk 1]
```diff
@@ -1,17 +1,50 @@
...
     APP.config['PROFILE'] = True
     APP.wsgi_app = ProfilerMiddleware(APP.wsgi_app, restrictions=[30])
 
-APP.run()
+if args.config:
+    config = args.config
+    if not config.startswith('/'):
+        here = os.path.join(os.path.dirname(os.path.abspath(__file__)))
+        config = os.path.join(here, config)
+    os.environ['PKGDB2_CONFIG'] = config
```
Comment (adrianreber) [inline runserver.py]: PKGDB2_CONFIG looks wrong

File: runserver.py [hunk 2]
```diff
@@ -1,17 +1,50 @@
...
+    '--host', default="127.0.0.1",
+    help='Hostname to listen on. When set to 0.0.0.0 the server is available \
+    externally. Defaults to 127.0.0.1 making the it only visable on localhost')
+
+args = parser.parse_args()
+
+if args.profile:
+    from werkzeug.contrib.profiler import ProfilerMiddleware
     APP.config['PROFILE'] = True
     APP.wsgi_app = ProfilerMiddleware(APP.wsgi_app, restrictions=[30])
```
Comment (adrianreber) [inline runserver.py]: APP is imported later. This will probably fail. Although, I don't know what and how the profile is used.

**Comment by adrianreber** [comment]
:+1:

**Comment by pypingou** [comment]
Thanks :)

**Pull request merged**